web-embeds/package.json

```diff
@@ -49,7 +49,7 @@
   },
   "scripts": {
     "eject": "react-scripts eject",
-    "start": "craco start",
+    "dev": "craco start",
```

Comment:
Does start also need to be supported for ci/cd?

Reply:
What do you mean? In Vercel?

Reply:
Yeah. Just wanna make sure we don't bork our deployment pipeline with a destructive change instead of an additive one. I'm not sure what's all required in the deployment process.

Reply:
Gotcha, yeah, I will need to jump into Vercel and update the start config 👍